### ISSUE TITLE
Editing the "about me" section

### ISSUE THREAD

**Issue by jkbdbrnwsk** [x-team/unleash#457]
Clicking on each row ("The Power of Your Dreams", tags, website) should make it editable. Only profile owners and Unleashers can edit the "about me" section and social media links.

Clicking on the social media icons, and input should appear where user can paste an appropriate link.

![image](https://cloud.githubusercontent.com/assets/7212882/26689835/5f636588-46f7-11e7-849d-f4fc07481dd2.png)

**Comment by jacekelgda** [comment]
I'd suggest rather using traditional form in settings section or profile section, this seems like unnecessary feature.

But if going with this flow, I would suggest showing separate edit button for each row or one for going into edit mode ( more difficult )

**Comment by jkbdbrnwsk** [comment]
Yeah, I like the separate "Profile settings" page idea. We can go with either, whatever you guys feel more comfortable with. :)
